packages/@aws-cdk/aws-certificatemanager/lambda-packages/dns_validated_certificate_handler/lib/index.js

```diff
@@ -77,10 +77,10 @@ let report = function(event, context, responseStatus, physicalResourceId, respon
  * @param {string} hostedZoneId the Route53 Hosted Zone ID
  * @returns {string} Validated certificate ARN
  */
-const requestCertificate = async function(requestId, domainName, subjectAlternativeNames, hostedZoneId, region) {
+const requestCertificate = async function(requestId, domainName, subjectAlternativeNames, hostedZoneId, region, route53Endpoint) {
   const crypto = require('crypto');
   const acm = new aws.ACM({ region });
-  const route53 = new aws.Route53();
+  const route53 = route53Endpoint ? new aws.Route53({endpoint: route53Endpoint}) : new aws.Route53();
```

Comment:
is it possible to provide the `region` when creating a new client? is it desirable to provide a full endpoint? It seems like the AWS SDK constructs this based on the configured [region](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Route53.html) 

> The endpoint URI to send requests to. The default endpoint is built from the configured region

Does that approach not work here?

Reply:
The template to be used to generate default endpoint is `{service}.{region}.amazonaws.com.cn`, please see the last comment of #6431 . And there is no such endpoint for route53 in China regions, so it does not solve the problem by providing a `region`. The currently working endpoint for route53 is `https://api.route53.cn` or `https://route53.amazonaws.com.cn`.